public/src/client/topic/postTools.js

```diff
@@ -338,7 +339,7 @@ define('forum/topic/postTools', [
 		}
 
 		if (post.length) {
-			slug = post.attr('data-userslug');
+			slug = utils.slugify(post.attr('data-username'), true);
```

Comment:
There might've been a reason we went with userslug instead of slugifying the username, but I cannot remember it now. Will need to properly test with utf-8 usernames for backwards compatibility.

Reply:
It worked with Gąska's username from TDWTF, which is about the only username I could think of that uses characters other than the Latin set. I suppose someone could create a set of other values to test with -- Cyrillic, Hebrew, Greek, Turkish, Japanese, etc.

I found where the slug was generated from the username and saved and was going to update it there, but I couldn't find where the slug was added to the post attributes and I didn't know whether a change there would affect somewhere else that expected all lowercase letters for the slug. And since this function appeared to be used only to get the slug when quoting/replying to another post, I figured regenerating the slug from the username again should be fine.

Reply:
> Gąska

Apart from [my test](https://what.thedailywtf.com/user/pjh_g%C4%85ska) to see if that would be acceptable on the software way back when before I renamed him, that is AFAIK, the only one we do have.

You could always create a couple... :D